### PR TITLE
Add scope input

### PR DESCRIPTION
**Pull request by whywaita**
This PR added `scope` to actions input.

`appOctokit.apps.listInstallations()` response array of installation. This array has multiple contents when installed to multi-organization or repositories.
So `scope` is a filter for an array of installation. 

`scope` inputs often the organization name or the repository name.

### CODE REVIEW

**Comment by whywaita** [comment]
@billyvg Hi, Can you review this?

**Comment by billyvg** [comment]
@whywaita Ah sorry, this got lost, will take a look now, thanks

File: src/main.ts [hunk 1]
```diff
@@ -1,28 +1,42 @@
-import {createAppAuth} from '@octokit/auth-app';
-import {Octokit} from '@octokit/rest';
-import * as core from '@actions/core';
+import { createAppAuth } from "@octokit/auth-app";
+import { Octokit } from "@octokit/rest";
+import { Endpoints } from "@octokit/types";
+import * as core from "@actions/core";
```
Comment (billyvg) [inline src/main.ts:4]: Oops, I guess I didn't run pre-commit hooks when I committed this 🙊🙊 - thanks!

File: src/main.ts [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
       auth: {
         id,
         privateKey,
       },
-      baseUrl: process.env['GITHUB_API_URL'] || 'https://api.github.com',
+      baseUrl: process.env.GITHUB_API_URL || "https://api.github.com",
     });
 
-    const {data} = await appOctokit.apps.listInstallations();
+    type listInstallationsResponse = Endpoints["GET /app/installations"]["response"];
```
Comment (billyvg) [inline src/main.ts:20]: Can you move this type alias out of the function? I think it's a bit confusing to have type aliases mixed inside of runtime code.

**Review by billyvg — COMMENTED** [review]
Left a note, but otherwise this looks good. Thanks!

File: src/main.ts (hunk 2 above)
Comment (whywaita) [reply src/main.ts]: I moved in https://github.com/getsentry/action-github-app-token/pull/11/commits/2652b2ad37d5d13d6ad7be4d31b4d94e2a35b36a

**Review by billyvg — APPROVED** [review]
Thanks again!

**Comment by whywaita** [comment]
Thank you!

**Pull request merged**